src/services/consumer.js

```diff
@@ -71,7 +71,7 @@ class Consumer {
                                 continue;
                             }
                             const pokestop = Pokestop.fromFort(fort.cell, fort.data);
-                            await pokestop.triggerWebhook(false);
+                            pokestop.triggerWebhook(false);
```

Comment:
```suggestion
                            await pokestop.triggerWebhook(false);
```
We need await for promises